docs/_data/prague-2022-schedule.yaml

```diff
@@ -0,0 +1,97 @@
+writing_day:
+- time: '10:00'
+  duration: '00:30'
+  title: 'Platform opens'
+- duration: '0:15'
+  title: Morning introduction - Introduction to Sprints & Project Introductions
+- duration: '2:15'
+  title: Team up and start working
+- duration: '1:00'
+  title: Lunch Break (60 mins)
+- time: '14:00'
+  duration: '0:30'
+  title: Conference Intro and Tour with the Welcome Wagon
+- time: '17:00'
+  title: End of Writing Day
+
+talks_day1:
+- time: '9:00'
+  duration: '0:30'
+  title: Platform opens
+- time: '9:30'
+  duration: '0:30'
+  title: Conference Intro and Tour with the Welcome Wagon
+- duration: '0:40'
+  title: Write the Docs Team - Introduction to Write the Docs
```

Comment:
Are we not aligning this with Portland's format? We shortened the intro and renamed it to "Community update" or something, and the next talk started at :30

Reply:
Also, I see that the talks are still listed in sets of 3 and no explicit Q&A. Did we decide to revert this?

Reply:
Urgh, I copied from Prague not from Portland